### PR TITLE
Add Telegram test

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

File: server/boot/create-default-tables.js [hunk 1]
```diff
@@ -220,6 +220,11 @@ module.exports = function(app) {
           'description': 'Checks to see if Facebook Messenger is working',
           'spec_url': 'https://github.com/TheTorProject/ooni-spec/blob/master/test-specs/ts-019-facebook-messenger.md'
       },
+          'name': 'telegram',
+          'long_name': 'Telegram',
+          'description': 'Checks to see if Telegram is working',
+          'spec_url': 'https://github.com/TheTorProject/ooni-spec/blob/master/test-specs/ts-020-telegram.md'
```
Comment (anadahz) [inline server/boot/create-default-tables.js]: This URL depends on https://github.com/TheTorProject/ooni-spec/pull/85 .

**Comment by hellais** [comment]
LGTM

File: server/boot/create-default-tables.js [hunk 2]
```diff
@@ -220,6 +220,11 @@ module.exports = function(app) {
           'description': 'Checks to see if Facebook Messenger is working',
           'spec_url': 'https://github.com/TheTorProject/ooni-spec/blob/master/test-specs/ts-019-facebook-messenger.md'
       },
+          'name': 'telegram',
```
Comment (hellais) [inline server/boot/create-default-tables.js]: I should really get used to testing things before merging them...

This contains invalid markup and leads to error.

I am going to revert this commit.

**Comment by hellais** [comment]
Can you please re-submit a pull request with valid syntax?

**Comment by anadahz** [comment]
Thanks for double-checking @hellais, I submitted a new pull request (#114).

**Pull request merged**